web/pandas_web.py

```diff
@@ -307,7 +307,10 @@ def roadmap_pdeps(context):
         with open(pathlib.Path(context["target_path"]) / "pdeps.json", "w") as f:
             json.dump(pdeps, f)
 
-        for pdep in sorted(pdeps["items"], key=operator.itemgetter("title")):
+        regex = "(?<=PDEP-)\\d+"
+        compiled_pattern = re.compile(regex)
+        get_num = lambda pdep: int(re.findall(compiled_pattern, pdep["title"])[0])
```

Comment:
I think `compiled_pattern.match(...)` should be more appropriate here.

Reply:
Yep you are right. Will change.